TestingApp/ViewController/ViewController.swift

```diff
@@ -265,9 +261,10 @@ extension ViewController {
 
         do {
             try Glia.sharedInstance.configure(
-                with: configuration, 
-                theme: Theme(),
-                uiConfig: uiConfig
+                with: configuration,
+                theme: theme,
+                uiConfig: uiConfig,
+                queueIds: [self.queueId]
```

Comment:
Would it be reasonable to use new method?

Reply:
Yes, will change that